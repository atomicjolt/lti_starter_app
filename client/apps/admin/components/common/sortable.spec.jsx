import React from 'react';
import TestRenderer from 'react-test-renderer';

import Sortable from './sortable';

describe('Sortable', () => {
  it('renders the Sortable component', () => {
    const result = TestRenderer.create(
        <Sortable />
    );
    expect(result).toMatchSnapshot();
  });
});
