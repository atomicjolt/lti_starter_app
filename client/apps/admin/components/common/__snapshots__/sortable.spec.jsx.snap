// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sortable renders the Sortable component 1`] = `
<th
  className="sortable-header"
>
  <div
    className="aj-flex"
  >
    <button
      className="aj-sort-btn"
      onClick={[Function]}
      type="button"
    >
      <span
        className="aj-flex selected"
      >
        <i
          className="material-icons aj-icon-btn-sort"
        >
          arrow_downward
        </i>
      </span>
    </button>
  </div>
</th>
`;
